### DISCUSSION TITLE
Postgres Boolean in Table & Date in Form

### DISCUSSION THREAD

**Discussion by andhie** [Budibase/budibase#2702] [Help]
I have postgres as external data source,
i have fields which is set to `boolean` data type in postgress
When its displayed in table, its displayed as `true` or `false`. my expectation would be a checkbox type (same as what we see in the Data > Sources > Tables). However in the Form view, it is able to correctly display as checkbox.

![image](https://user-images.githubusercontent.com/3997169/134224738-9bee0a2a-eee8-4013-bce7-e61e86c9b93b.png)

For the second case, i have a date field which is a `date` data type in postgres,
When displaying it in Form view, it is just a plain text field. my expectation is should have a calendar popup (similar when using internal db)

![image](https://user-images.githubusercontent.com/3997169/134226041-1ded4878-77bc-48c2-92a1-53eff48647be.png)

Are these the current limitations or is there a way to change it?

**Comment by mjashanks** [comment]
Hey  Andhie. This is a logged issue: https://github.com/Budibase/budibase/issues/2642

It works if your postgres column is a `timestamp`. I appreciate that it might not be possible to change it. Hope to get this fixed soon.
